src/app/encoder.cpp

```diff
@@ -173,4 +173,65 @@ uint16_t encodeReadOnOffCommand(uint8_t * buffer, uint16_t buf_length, uint8_t d
                                        1 /* attr_id_count */);
 }
 
+uint16_t _encodeIdentifyClusterCommand(uint8_t * buffer, uint16_t buf_length, uint8_t command_id, uint8_t destination_endpoint,
+                                       uint16_t * identify_duration)
+{
+    uint32_t result = 0;
+    // pick cluster id as 3 for now.
+    // pick source and destination end points as 1 for now.
```

Comment:
We're not setting the destination endpoint to 1... it's passed in.  I guess `_encodeOnOffCommand` has the same bogus comment, sorry.

We should really share more code here with the on/off code.  Please file a followup for that?

Reply:
I did a preemptive PR to fix that in #2766